client.go

```diff
@@ -97,7 +97,7 @@ func (c *client) executeRequest(ctx context.Context, req *internalRequest) error
 		Function:         req.functionName,
 		RequestToString:  "empty request",
 		ResponseToString: "empty response",
-		MeilisearchApiError: meilisearchApiError{
+		MeilisearchApiError: APIError{
```

Comment:
can you remove that from the PR? Can you remove everything that is not related to the original PR which mean moving tests?